hilfe/handbuch/einstellungen/zuschläge/index.md

```diff
@@ -0,0 +1,108 @@
+
+>**Achtung!** Verändern Sie die grundangelegten SFN-Zuschlagsarten nur, wenn Sie wissen, was Sie tuen - Veränderungen an diesen können schnell in Lohnverstößen enden!
+
+## Neue Zuschlagsart hinzufügen
+
+Klicken sie auf {% button icon = "plus", label = "Neue Zuschlagsart"%} um eine neuen Zuschlagsart zu erstellen. Anschließend öffnet sich ein Bearbeitungsdialog in welchem Sie die Eigenschaften des Zuschlags definieren können. 
+
+### Name der Zuschlagsart
+
+Geben Sie hier den Namen der Zuschlagsart ein.
+
+### Gültig an
+
+Hier können Sie auswählen, an welchen Tage der Zuschlag zu Wirkung kommen soll. 
+
+#### Täglich
+
+Der Zuschlag wird allen ausgewählten Mitarbeitern jeden Tag im ausgewählten Zeitraum angerechnet. **Nützlich bspw. für Nachtschichten.**
+
+#### Bestimmte Wochentage
+
+Der Zuschlag wird allen ausgewählten Mitarbeitern an den ausgewählten Tagen angerechnet. Wählen Sie dazu einfach die gewünschten Wochentage aus. 
+
+{% figure caption= "Dieser Zuschlag wird den Mitarbeitern nur Samstags von 20 Uhr bis 4 Uhr des foglenden Tages angerechnet."%}
+<img src = "samstagnacht.webp">
+{% endfigure %}
+
+#### Feiertage
+
+Sollten Sie einen speziellen Zuschlagssatz für bestimmte Feiertage gewähren wollen, können Sie dies hier tun. Wählen Sie einfach die gewünschten Feiertage aus der Liste aus. 
+
+#### Datum 
+
+Vielleicht ist der Tag den Sie auswählen wollen gar kein offizieller Feiertag bspw. Neujahr. Unter **Datum** können Sie jeden Tag des Jahres auswählen. 
+
+### Zeiträume
+
+Hier können Sie die Stunden auswählen zu denen der Zuschlag gewährt werden soll. Klicken Sie auf {%icon "plus" %} und {% icon "minus" %} um zusätzliche Zeitspannen hinzuzufügen. Ziehen Sie anschließend die Leisten so mittels Drag&Drop, dass sie den gewünschten Zeitraum beinhalten. 
+
+### Prozent Maximal
+
+Der hier eingetragene Wert gilt als Obergrenze wenn diese Zuschlagsart einem Mitarbeiter zugeteilt wird. Dies verhindert, dass Sie ausversehen einen zu hohen Betrag im Vertrag eines Mitarbeiters angeben. 
+
+{% figure caption= "Wenn sich mal eine null zu viel einschleicht hilft Ihnen der voreingestellte maximale Prozentsatz."%}
+<img src ="maximale_prozent.webp">
+{% endfigure %}
+
+### Prozent Standardwert
+
+Dieser Wert ist die Norm, die beim Auswählen dieser Zuschlagsart für einen Mitarbeiter ausgewählt wird.
+
+### Beitragsfrei bis zu
+
+Hier wird bestimmt ob der Zuschlag betragsfrei abgerechnet werden kann. **Generell gilt, dass nur gesetzlich verpflichtete SFN-Zuschläge bis zu 25€/h beitragsfrei sind.** Sollte es sich um einen SFN-Zuschlag handeln haben Sie normalerweise natürlich Interesse daran diesen so lange wie möglich beitragsfrei abzurechnen. Deshalb sollte dieses Feld normalerweise mit 25€/h für gesetzlich verpflichtende und 0€/h für restliche Zuschläge ausgefüllt werden. 
```

Comment:
```suggestion
Hier wird bestimmt ob der Zuschlag betragsfrei abgerechnet werden kann. **Generell gilt, dass nur gesetzlich [gesetzlich definierte SFN-Zuschläge](https://www.gesetze-im-internet.de/estg/__3b.html) bis zu 25€/h beitragsfrei sind.** Sollte es sich um einen SFN-Zuschlag handeln haben Sie normalerweise natürlich Interesse daran diesen so lange wie möglich beitragsfrei abzurechnen. Deshalb sollte dieses Feld normalerweise mit 25 € / Std. für SFN-Zuschläge und 0 € / Std. für restliche Zuschläge ausgefüllt werden. 
```

Achtung: SFN-Zuschläge sind **nicht** "gesetzlich verpflichtend"! Ob Zuschläge gezahlt werden oder nicht und in welcher Höhe, darf der Arbeitgeber schon frei entscheiden (außer es ist im Tarifvertrag definiert).